src/rgw/support/rgw_support_bundle

```diff
@@ -166,8 +161,35 @@ class RGWSupportBundle:
         if coredumps:
             crash_dump_dir = os.path.join(log_dir, 'rgw_debug')
             if os.path.exists(crash_dump_dir):
-                shutil.copytree(crash_dump_dir,\
-                    os.path.join(RGWSupportBundle._tmp_src, 'rgw_debug'))
+
+                if size_limit:
```

Comment:
if size_limit is passed, pick last 5 cores, else pick all

Reply:
core files are heavy, keep max 2 by default. and also compare core file size with that of size_limit, and if two cant be accommodated, collect only 1 atleast, even if its going beyond given size limit